compiler_gym/envs/llvm/service/ObservationSpaces.h

```diff
@@ -46,6 +46,58 @@ enum class LlvmObservationSpace {
    *     deep reinforcement learning. FCCM.
    */
   AUTOPHASE,
+  /**
+   * The IR2Vec Program Level Flow-Aware embeddings.
+   *
+   * From:
+   *
+   * 	  S. VenkataKeerthy, Rohit Aggarwal, Shalini Jain, Maunendra Sankar Desarkar,
+                                Ramakrishna Upadrasta, and Y. N. Srikant. (2020).
+                                IR2VEC: LLVM IR Based Scalable Program Embeddings.
+                                ACM Trans. Archit. Code Optim. 17, 4, Article 32 (December 2020), 27
+   pages. DOI:https://doi.org/10.1145/3418463
+   *
+   */
+  IR2VEC_FA,
```

Comment:
Could you use full names rather than abbreviations?

```
IR2VEC_FLOW_AWARE
IR2VEC_SYMBOLIC
IR2VEC_FUNCTION_LEVEL_FLOW_AWARE
IR2VEC_FUNCTION_LEVEL_SYMBOLIC
```

It is easier to understand:
```py
env = gym.make("llvm-v0", observation_space="Ir2vecFlowAware")
```
than
```py
env = gym.make("llvm-v0", observation_space="Ir2vecFa")
```

Reply:
Done!